src/south/sonic/goldstone/south/sonic/portchannel.py

```diff
@@ -134,6 +175,15 @@ def oper_cb(self, xpath, priv):
             members = self.sonic.get_keys(f"LAG_MEMBER_TABLE:{name}:*", "APPL_DB")
             members = [m.split(":")[-1] for m in members]
             state["interface"] = members
+            state["interface"] = members
+            static = self.sonic.sonic_db.get(
+                self.sonic.sonic_db.CONFIG_DB, f"PORTCHANNEL|{name}", "static"
+            )
+            if static is not None and static == "true":
```

Comment:
```
if static == "true":
```

should be enough?

Reply:
Thank you for your comment. This optimization has been done in the latest commit.